File: e2e/python/comments.py
# Python comments for testing

# This is a single-line comment
value = 42

"""
This is a multi-line comment
with multiple lines of text
"""

def test():
    # Another comment inside function
    return value

def greet(name):
    """
    Documentation string for greet function
    Args:
        name: The name to greet
    Returns:
        A greeting message
    """
    return f"Hello, {name}!"

class Example:
    """Example class for testing"""
    
    def method(self):
        """Method documentation"""
        pass

File: e2e/go/comments.go
// Go to Test Reviews
package main

// This is a one-line comment
const value = 42

/*
This is a multi-line comment
Multi-line text
*/

func test() int {
    // Another comment inside the function
    return value
}

// greet returns a greeting message
// name is the person to greet
func greet(name string) string {
    return fmt.Sprintf("Hello, %s!", name)
}
